src/Notifier.php

```diff
@@ -558,15 +562,21 @@ protected function errorNotifications()
 
     protected function remoteErrorConfig()
     {
-        if (isset($this->errorConfig)) {
-            return $this->errorConfig;
+        if ($this->opt['remoteConfig'] == false) {
+            return RemoteConfig::DEFAULT_CONFIG;
         }
 
-        $remoteConfig = new RemoteConfig($this->opt['projectId']);
+        $remoteConfig = $this->newRemoteConfig($this->opt['projectId']);
         $this->errorConfig = $remoteConfig->errorConfig();
         return $this->errorConfig;
     }
 
+    protected function newRemoteConfig($projectId)
```

Comment:
my main motivation for having a `newRemoteConfig` method is that it made things easier to mock in tests. 

We can remove the `$remoteConfig` assignment here though, sorry that was leftover from some local debugging. The body of this method should just be:
```php
   return new RemoteConfig($projectId);
```

Since the `$remoteConfig` variable isn't prefixed with `$this->` it will only be available in the method scope and not on the instance.